crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -608,16 +608,7 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
                 EventStatus::Failed(errmsg.into())
             }
             Ok(Ok(())) => {
-                if let Some((tx_data, bytes_written)) = stdb.commit_tx(&ctx, tx).unwrap() {
-                    // TODO(cloutiertyler): This tracking doesn't really belong here if we want to write transactions to disk
-                    // in batches. This is because it's possible for a tiny reducer call to trigger a whole commit to be written to disk.
-                    // We should track the commit sizes instead internally to the CommitLog probably.
-                    if let Some(bytes_written) = bytes_written {
-                        WORKER_METRICS
-                            .reducer_write_size
-                            .with_label_values(&address, reducer_name)
-                            .observe(bytes_written as f64);
-                    }
+                if let Some(tx_data) = stdb.commit_tx(&ctx, tx).unwrap() {
```

Comment:
Note that it is no longer straightforward to report this metrics due to the durability layer being asynchronous and meant to buffer transaction data.

It is surely possible, however, to calculate the size of `tx_data` as it would be written to the log ... eventually. It's not clear if this is what we want, though.

/cc @joshua-spacetime @cloutiertyler

Reply:
@kim If I'm not mistaken, I don't think this metric is even being used. So I think it's fine to delete.